CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
-If you find a bug or you'd like to propose a feature, please feel free to raise an issue on our [issue tracker](https://github.com/input-output-hk/cardano-wallet/issues).
+If you find a bug or you'd like to propose a feature, please feel free to raise an issue on our [issue tracker](https://github.com/cardano-foundation/cardano-wallet/issues).
 
 Pull requests are welcome!
 
 When creating a pull request, please make sure that your code adheres to our [coding standards](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md).
```

Comment:
Note that this link should not be changed until we relocate our coding standards document. This will be tackled as part of https://cardanofoundation.atlassian.net/browse/ADP-2256.